tox.ini

```diff
@@ -19,3 +19,28 @@ deps =
     yamllint==1.15.0
 commands =
     bash -c "yamllint eve/main.yml salt/**/*.yaml"
+
+[testenv:ci-tests]
+deps =
+    pytest
+    pytest_bdd
+    testinfra
+    paramiko
+    cryptography==2.4.2
+commands =
+    pytest --ssh-config=some/identity/files --hosts=[list, of, hosts] -k "ci" tests
+
+[testenv:local-tests]
+setenv =
+    VAGRANT_CWD = {toxinidir}
+commands_pre =
+    bash -c "vagrant up"
```

Comment:
With this setup, the test env is not the env one gets from `make vagrantup`, which can be confusing.

I think we'd better have the tests *not* set up the Vagrant env (but test its status before launch, in the Tox config), and only execute against a `make vagrantup` environment, which is the same as the one a user gets when running `vagrant ssh`, for debugging etc.